pkg/kubelet/apis/config/types.go

```diff
@@ -335,6 +335,11 @@ type KubeletConfiguration struct {
 	// This provide a "static" CPU list rather than the "dynamic" list by system-reserved and kube-reserved.
 	// This option overwrites CPUs provided by system-reserved and kube-reserved.
 	ReservedSystemCPUs string
+
+	// These options specifies the ratio to oversold node resources.
+	// If 0, these fields are ignored.
+	CPUOvercommitFactor uint
```

Comment:
If this is a ratio, why is it an integer? You appear to only be allowing 2x, 3x, 4x, etc resources advertised.

Reply:
Thx riking. Ohoh, due to cpuset, we only support 2x, 3x, 4x. Or I change ratio to a better name?

Reply:
I would think that you use a different OCI interface than cpusets if you're oversubscribing..?